nemo/collections/asr/parts/jasper.py

```diff
@@ -36,6 +37,28 @@
 jasper_activations = {"hardtanh": nn.Hardtanh, "relu": nn.ReLU, "selu": nn.SELU, "swish": Swish}
 
 
+def tds_uniform_(tensor, mode='fan_in'):
+    """
+    """
+    fan = _calculate_correct_fan(tensor, mode)
+    gain = 2.0
+    std = gain / math.sqrt(fan)  # sqrt(4.0 / fan_in)
+    bound = std  # Calculate uniform bounds from standard deviation
+    with torch.no_grad():
+        return tensor.uniform_(-bound, bound)
+
+
+def tds_normal_(tensor, mode='fan_in'):
+    """
+    """
+    fan = _calculate_correct_fan(tensor, mode)
+    gain = 2.0
```

Comment:
Can these initialization be useful for others? If yes, you may move them somewhere where is shared?

Reply:
Hmm they are somewhat specific to Jasper based models right now (ill add a reference to the paper). We can move them to commons if anyone needs it in the future.